test/unit/controllersSpec.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+/* jasmine specs for controllers go here */
+describe('githubCreep controllers', function() {
+
+  describe('userListCtrl', function(){
+    var scope, ctrl, $httpBackend;
+
+    beforeEach(module('githubCreep'));
+    beforeEach(inject(function(_$httpBackend_, $rootScope, $controller) {
+      $httpBackend = _$httpBackend_;
+      $httpBackend.expectGET('users/users.json').respond('Hello FU');
+
+      scope = $rootScope.$new();
+      ctrl = $controller('userListCtrl', {$scope: scope});
+    }));
+
+
+    it('stores result of API call in users', function() {
+      expect(scope.users).toBeUndefined();
+      $httpBackend.flush();
+
+      expect(scope.users).toEqual([
+            {
+              "login": "drjparry",
+              "avatar_url": "https://avatars.githubusercontent.com/u/13313921?v=3"
+            }]);
+    });
+
+
+  });
+});
+
+
+
+
+// 'use strict';
```

Comment:
Such screaming :scream_cat: - no need for commented code, it should all be visible in your commit history if people want it.